web/packages/teleport/src/services/agents/types.ts

```diff
@@ -90,6 +90,16 @@ export type ResourceIdKind =
   | 'windows_desktop'
   | 'saml_idp_service_provider';
 
+export enum ResourceIdKinds {
```

Comment:
This enum mirrors the type that we have above. I'd avoid doing this, adding a new type would require changing it in both places. Additionally, we no longer have a single source of truth.

Instead, I'd use only `ResourceIdKind`. When it comes to `encodeUrlQueryParams`, you can create a specialized version in `ResourcesSections.tsx` that accepts only `ResourceIdKind` as `kind`. 
Ideally, we would just change `encodeUrlQueryParams` signature, but this will likely break other places.

Reply:
I think we can remove it now?